chore(docs): Update Docs for DevDot Migration (#2034)
hashicorp/terraform-cdk: website/docs/cdktf/concepts/constructs.mdx

## Use Constructs

> > **Hands On:** Try [Deploy Applications with CDK for Terraform](https://learn.hashicorp.com/tutorials/terraform/cdktf-applications?in=terraform/cdktf) tutorial to use a custom construct. It includes the example code below.

Reviewer: ```suggestion
> **Hands On:** Try [Deploy Applications with CDK for Terraform](https://learn.hashicorp.com/tutorials/terraform/cdktf-applications?in=terraform/cdktf) tutorial to use a custom construct. It includes an example code.
```

Suggest removing "below" as a directional word.

Author: yeah i'm going to take out that entire line honestly. I don't think it's required here!
